pkg/cli/admin/prune/images/images.go

```diff
@@ -753,7 +755,12 @@ func getRegistryClient(clientConfig *restclient.Config, registryCABundle string,
 	}
 
 	// we have to set a username to something for the Docker login but it's not actually used
-	registryClientConfig.Username = "unused"
+	buffer := make([]byte, 5)
+	_, err = rand.Read(buffer)
+	if err != nil {
+		return nil, err
+	}
+	registryClientConfig.Username = base64.URLEncoding.EncodeToString(buffer)[:5]
```

Comment:
this might be simpler to use 

https://github.com/openshift/oc/blob/48834dcb1646a119859c14a51c10cecea4bf58bc/pkg/cli/debug/debug.go#L459

Reply:
Ah, that's great. I wasn't happy with the current string generation in this PR :). Let me update with this now.

Reply:
yes, no need to use MakeSimpleName. Updated to only use utilrand.